--- src/term.c.orig	2010-12-17 09:59:37.000000000 +0100
+++ src/term.c	2010-12-17 10:02:21.000000000 +0100
@@ -250,7 +250,7 @@
      desc.flags  = DFDESC_HEIGHT;
      desc.height = fontsize;
 
-     ret = dfb->CreateFont (dfb, LITEFONTDIR"/Misc-Fixed.pfa", &desc, &font);
+     ret = dfb->CreateFont (dfb, "/usr/share/LiTE/fonts/Misc-Fixed.pfa", &desc, &font);
      if (ret) {
           lite_close();
           dfb->Release (dfb);
